packages/json-schemas/schemas/ec_signature_schema.json

```diff
@@ -1,14 +1,4 @@
 {
     "id": "/ecSignatureSchema",
-    "properties": {
-        "v": {
-            "type": "number",
-            "minimum": 27,
-            "maximum": 28
-        },
-        "r": { "$ref": "/ecSignatureParameterSchema" },
-        "s": { "$ref": "/ecSignatureParameterSchema" }
-    },
-    "required": ["v", "r", "s"],
-    "type": "object"
+    "type": "string"
```

Comment:
the structure of `ecSignatureSchema` had been changed from (v,r,s) object to string